src/icons/downArrow.js

```diff
@@ -0,0 +1,3 @@
+const icon = '<svg width="8px" height="5px" viewBox="0 0 8 5" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs></defs><g id="Sortable-headers" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g id="Nearby-inventory" transform="translate(-680.000000, -249.000000)" fill="#252729"><polygon id="ic_downarrow" transform="translate(684.000000, 251.500000) scale(1, -1) translate(-684.000000, -251.500000) " points="684 249 688 254 680 254"></polygon></g></g></svg>';
```

Comment:
It looks like we're duplicating icon content (svg markup) in this .js file and in .svg file below (as well as for an up arrow). Do we need svg file then? Can we only have one?